src/createServer.js

```diff
@@ -29,12 +31,13 @@ const createServer = ({
   secret,
   development = true,
   log = true,
+  logger = morgan(development ? 'dev' : 'common'),
```

Comment:
Could we make this just the `log` option? So `false` disables logging, `true` uses the default `morgan` logger, and anything else will be passed directly into `server.use`?